### ISSUE TITLE
Keyboard shortcuts

### ISSUE THREAD

**Issue by Kitzberger** [threema-ch/threema-web#37]
Would be nice to be able to control the application by keyboard only. E.g. switching conversations by pressing a handy shortcut, like in slack via [ctrl]+[k]

**Comment by dbrgn** [comment]
I forgot that we already had this issue and opened #172.

Closing this one since the other ticket already has more comments. Sorry :)
